src/NuGet.Clients/NuGet.PackageManagement.UI/PackageLoadContext.cs

```diff
@@ -31,6 +31,8 @@ internal class PackageLoadContext
 
         public PackageSearchMetadataCache CachedPackages { get; set; }
 
+        public IVsSolutionManager SolutionManager { get; private set; }
```

Comment:
Not in all cases, we usually do it for MEF components or VS packages. Using ServiceLocator will unnecessary add a perf overhead here.